AmplifyPlugins/Auth/Tests/AuthHostApp/AuthIntegrationTests/SignInTests/AuthSRPSignInTests.swift

```diff
@@ -318,7 +318,7 @@ class AuthSRPSignInTests: AWSAuthBaseTest {
             signInExpectation.fulfill()
         }
 
-        wait(for: [signInExpectation, fetchAuthSessionExpectation], timeout: networkTimeout)
+        wait(for: [signInExpectation, fetchAuthSessionExpectation], timeout: 10)
```

Comment:
There are two apis working in parallel, the default 5 second timeout might fail for slower network connections.